powerauth-java-client-spring/src/main/java/io/getlime/security/powerauth/soap/spring/client/PowerAuthServiceClient.java

```diff
@@ -208,6 +237,31 @@ public CommitActivationResponse commitActivation(String activationId, String ext
         return this.commitActivation(request);
     }
 
+    /**
+     * Call the updateActivationOtp method of PowerAuth 3.1 Server SOAP interface.
+     * @param activationId Activation ID for activation to be updated.
+     * @param externalUserId User ID of user who updated the activation. Use null value if activation owner caused the change,
+     *                       or if OTP value is automatically generated.
```

Comment:
Strange formatting.

Reply:
Newlines in `@param` are regular way. I just added more indentation to both params.